packages/react-shopify-app-bridge/spec/Provider.spec.tsx

```diff
@@ -103,9 +104,6 @@ describe("GadgetProvider", () => {
     });
 
     test("can render a standalone app type in embedded context", () => {
-      // @ts-expect-error mock
-      jest.spyOn(window, "self", "get").mockImplementation(() => ({}));
```

Comment:
instead of relying on `window.self === window.top` app bridge can now tell us if we're embedded

Reply:
woooo!